### DISCUSSION TITLE
stable compact binary format for exporting database

### DISCUSSION THREAD

**Discussion by danz68** [duckdb/duckdb#5361] [Ideas]
I am not sure if this was mentioned, but right now, for large databases, EXPORT DATABASE creates huge CSV's, right? Why not using PARQUET? Or any other stable binary format you may consider?

**Comment by Mytherin** [comment]
You can pass `FORMAT PARQUET` to `EXPORT DATABASE`: 

```sql
EXPORT DATABASE 'export_test' (FORMAT PARQUET)
```

The default is `CSV` for portability reasons.

**Reply by danz68** [reply]
I was just about to come and remove the comment, because I just went through documentation again and saw just that!
Thank you!
